modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPDefinitionLocalServiceImpl.java

```diff
@@ -1406,6 +1407,12 @@ public CPDefinition deleteCPDefinition(CPDefinition cpDefinition)
 			companyGroup.getGroupId(), CProduct.class,
 			cpDefinition.getCProductId());
 
+		// Commerce product configuration entries
+
```

Comment:
this is fine, but we need to also add a delete method that can't delete entries of a master configuration list